stl/inc/yvals_core.h

```diff
@@ -855,6 +855,42 @@
 
 // next warning number: STL4027
 
+// P0619R4 remove deprecated features from C++20

```

Comment:
I would prefer following #28's spelling of `P0619R4 Removing C++17-Deprecated Features In C++20` here.

Reply:
yeah